index.js

```diff
@@ -31,6 +32,6 @@ module.exports = [
     'last 2 firefox versions',
     'last 2 ios versions',
     'last 2 safari versions',
-    'last 2 edge versions',
+    'last 1 edge version',
```

Comment:
You should still keep the last 2 version but if you update the version `browserslist` the supported version range will get updated